src/renderer/screens/config/ConfigScreens/WorkspaceScreen.js

```diff
@@ -120,9 +122,43 @@ class WorkspaceScreen extends Component {
           <h4>TRUFFLE PROJECTS</h4>
           <div className="Row">
             <div className="RowItem">
+              {this.props.config.validationErrors["workspace.project"] && (
+                <div>
+                  <div className="ValidationError">
+                    {
+                      this.props.config.validationErrors["workspace.project"]
+                        .message
+                    }
+                    <button
+                      onClick={() => {
+                        this.setState({
+                          showErrorDetails: !this.state.showErrorDetails,
+                        });
+                      }}
+                    >
+                      <OnlyIf test={this.state.showErrorDetails}>
+                        hide stack trace
+                      </OnlyIf>
+                      <OnlyIf test={!this.state.showErrorDetails}>
+                        show stack trace
+                      </OnlyIf>
+                    </button>
+                    <OnlyIf test={this.state.showErrorDetails}>
+                      <div className="ValidationDetails">
+                        <SyntaxHighlighter language="bash">
+                          {this.props.config.validationErrors[
+                            "workspace.project"
+                          ].stack.map(line => line.toString())}
+                        </SyntaxHighlighter>
+                      </div>
+                    </OnlyIf>
```

Comment:
Again, avoid use of the `<OnlyIf>` component. You can use short-circuiting here:

```js
{this.state.showErrorDetails && <div>...</div>}
```